tests/integration/api/queries/shop/shopBySlug.test.js

```diff
@@ -0,0 +1,48 @@
+import { encodeShopOpaqueId } from "@reactioncommerce/reaction-graphql-xforms/shop";
+import TestApp from "/imports/test-utils/helpers/TestApp";
+
+jest.setTimeout(300000);
+
+let shopBySlagQuery;
```

Comment:
```suggestion
let shopBySlugQuery;
```

Reply:
I was looking at this for several seconds trying to figure out what was the proposed change and at the end I had to use https://text-compare.com/  🙃

Thanks Machico!